packages/d/devextreme.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "devextreme",
+  "description": "HTML5 JavaScript Component Suite for Responsive Web Development",
+  "keywords": [
+    "html5",
+    "controls",
+    "widgets",
+    "components",
+    "jquery",
+    "knockout",
+    "angular",
+    "web",
+    "ui",
+    "datagrid",
+    "pivotgrid",
+    "scheduler",
+    "treelist",
+    "chart"
+  ],
+  "homepage": "https://js.devexpress.com/",
+  "license": "https://js.devexpress.com/Licensing/",
+  "authors": [
+    {
+      "name": "Developer Express Inc.",
+      "email": "management@devexpress.com",
+      "url": "https://www.devexpress.com/"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/DevExpress/DevExtreme.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "devextreme",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "js/*.js",
+          "js/localization/*.js",
+          "js/vectormap-data/*.js",
+
+          "css/*.css",
+          "css/fonts/*.ttf",
+          "css/fonts/*.woff",
+          "css/fonts/*.woff2",
+          "css/icons/*.ttf",
+          "css/icons/*.woff",
+          "css/icons/*.woff2"
```

```suggestion
          "css/@(fonts|icons)/*.@(ttf|woff|woff2)"
```
We can reduce this down to a single glob

[Glob](https://www.digitalocean.com/community/tools/glob?comments=true&glob=css%2F%40%28fonts%7Cicons%29%2F%2A.%40%28ttf%7Cwoff%7Cwoff2%29&matches=false&tests=css%2Ffonts%2Ftest.ttf&tests=css%2Ffonts%2Ftest.woff&tests=css%2Ffonts%2Ftest.woff2&tests=css%2Ffonts%2Ftest.bad&tests=css%2Ficons%2Ftest.ttf&tests=css%2Ficons%2Ftest.woff&tests=css%2Ficons%2Ftest.woff2&tests=css%2Ficons%2Ftest.bad)